README.md

```diff
@@ -25,9 +25,20 @@ There are two endpoints. The default takes JSON as input, encrypts it and places
 
 Some of important environment variables available for configuration are listed below:
 
-| Environment Variable  | Default     | Description
-|-----------------------|-------------|----------------
-| FTP_HOST              | `pure-ftpd` | FTP to monitor
-| FTP_USER              | _none_      | User for FTP account if required
-| FTP_PASS              | _none_      | Password for FTP account if required
-| ENABLE_EMPTY_FTP      | `0`         | `1=on,0=off` Enables the ability to auto-empty the target FTP - **SHOULD NOT BE SWITCHED ON IN A PRODUCTION ENVIRONMENT!**
+| Environment Variable    | Default                               | Description
+|-------------------------|---------------------------------------|----------------
+| FTP_HOST                | `pure-ftpd`                           | FTP to monitor
+| FTP_USER                | _none_                                | User for FTP account if required
```

Comment:
Consistency with backticks in Default column needed - what's the rationale for leaving some off, if any?